modules/local/combine_uniprot.nf

```diff
@@ -4,7 +4,7 @@ process COMBINE_UNIPROT {
     conda "conda-forge::sed=4.7"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
-        'ubuntu:20.04' }"
+        'docker.io/ubuntu:20.04' }"
```

Comment:
At the end it has to be this:
```suggestion
        'docker.io/library/ubuntu:20.04' }"
```
See [here](https://github.com/nf-core/chipseq/pull/345#discussion_r1185749080)